bower.json

```diff
@@ -40,12 +40,16 @@
     "package.json",
     "public"
   ],
+  "devDependencies": {
```

Comment:
Yeah, they should only show up when you run ``bower install`` from this repo. If you were in edX web certs or another place using this repo's bower package, they would not be installed alongside all of the items in the ``dependencies`` list. Nifty for local shiz.